service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java

```diff
@@ -1,5 +1,6 @@
 package com.codeforcommunity.processor;
 
+import static org.jooq.generated.Tables.*;
```

Comment:
Can you change this back to individual imports? You can disable * imports in the settings as well if it keeps happening